stylesheets/styles.css

```diff
@@ -2100,4 +2100,4 @@ ul.legals li:last-child::after {
 
 #modal-10 li{
 	margin-bottom:20px;
-}
\ No newline at end of file
+}
```

Comment:
Remove this change and squash your commits into one.

Reply:
removed the extra changes @divyam3897